kitsune/sumo/parser.py

```diff
@@ -288,6 +290,18 @@ def _parse(locale):
 
         return html
 
+    def _abbrTagHook(self, parser, body, attributes):
```

Comment:
We should stick to the Python recommended snake case in our code, even though the `py-wikimarkup` package doesn't.
```suggestion
    def _abbr_tag_hook(self, parser, body, attributes):
```